### PR TITLE
clarified the possibility to change the configuration

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6962

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/mac-preferences.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ ms.topic: conceptual
 
 ## Summary
 
-In enterprise organizations, Microsoft Defender ATP for Mac can be managed through a configuration profile that is deployed by using one of several management tools. Preferences that are managed by your security operations team take precedence over preferences that are set locally on the device. Users in your organization are not able to change preferences that are set through the configuration profile.
+In enterprise organizations, Microsoft Defender ATP for Mac can be managed through a configuration profile that is deployed by using one of several management tools. Preferences that are managed by your security operations team take precedence over preferences that are set locally on the device. Changing of the preferences that are set through the configuration profile would require escalated privileges and would not be available for users wihtout administrative permissions.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-preferences.md:34]:
```suggestion
In enterprise organizations, Microsoft Defender ATP for Mac can be managed through a configuration profile that is deployed by using one of several management tools. Preferences that are managed by your security operations team take precedence over preferences that are set locally on the device. Changing the preferences that are set through the configuration profile requires escalated privileges and is not available for users without administrative permissions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
Hi @Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**